mod-source-record-manager-server/src/main/java/org/folio/services/util/ParsedRecordUtil.java

```diff
@@ -0,0 +1,36 @@
+package org.folio.services.util;
+
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+import org.folio.rest.jaxrs.model.ParsedRecord;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static org.apache.commons.lang3.StringUtils.EMPTY;
+import static org.apache.commons.lang3.StringUtils.SPACE;
+
+public final class ParsedRecordUtil {
+
+  public static final String SUBFIELDS_DATA_DELIMITER = SPACE;
+
+  private ParsedRecordUtil() {
+  }
+
+  public static String retrieveDataByField(ParsedRecord parsedRecord, String fieldTag, List<String> subfieldCodes) {
```

Comment:
Let`s add some docs for this Util/method.

Reply:
it has already been added in the previous commit)